### PR TITLE
Feature/position card timeline

### PR DESCRIPTION
**Pull request by eskil0312**

### CODE REVIEW

File: src/components/org/PositionCard/components/TimelineInstance.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React, { useMemo } from 'react';
+import styles from '../styles.less';
+import { PositionInstance } from '@equinor/fusion';
+import { useTooltipRef } from '@equinor/fusion-components';
+import classNames from 'classnames';
+
+type TimelineInstanceProps = {
+    instance: PositionInstance;
+    currentInstance: PositionInstance | null;
```
Comment (MartinForre) [inline src/components/org/PositionCard/components/TimelineInstance.tsx]: What's the difference between `instance` and `currentInstance`?

File: src/components/org/PositionCard/components/TimelineInstance.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    calculator: (time: number) => number;
```
Comment (MartinForre) [inline src/components/org/PositionCard/components/TimelineInstance.tsx]: The position calculator is really an internal calculation that should belong to the TimelineInstance component, not the consumer

File: src/components/org/PositionCard/components/TimelineInstance.tsx (hunk 1 above)
Comment (eskil0312) [reply src/components/org/PositionCard/components/TimelineInstance.tsx]: instance is the instance from the frominstances map(), currentInstance is the current

File: src/components/org/PositionCard/components/TimelineInstance.tsx (hunk 2 above)
Comment (eskil0312) [reply src/components/org/PositionCard/components/TimelineInstance.tsx]: Moved calculator

File: src/components/org/PositionCard/components/TimelineInstance.tsx (hunk 1 above)
Comment (MartinForre) [reply src/components/org/PositionCard/components/TimelineInstance.tsx]: Could the naming of these be changed to be less ambiguous?

Comment (eskil0312) [reply src/components/org/PositionCard/components/TimelineInstance.tsx]: Adde jsDoc

**Pull request merged**